packages/lit-element/src/lit-element.ts

```diff
@@ -170,3 +170,13 @@ if (DEV_MODE) {
     return true;
   };
 }
+
+export const $private = {
```

Comment:
Few questions:

* Why is this here and not in UpdatingElement since `_$attributeToProperty` is defined there? Is it because we only support SSR for LitElement?
* What does the leading `$` indicate here? 
* If this is for SSR, should it be named as such?

Reply:
OK, I understand this based on the pattern done in lit-html

Reply:
Background: This is following the patterns already established in `lit-html`

1. Yes, SSR only for LitElement
2. Just extra warning to users that they shouldn't rely on fields in it. Open to other suggestions.
3. We can use a different name; since it's a top-level export, the longer the name the longer

Reply:
I updated it to `_$private`